lib/Ama.pm

```diff
@@ -84,10 +88,14 @@ $self->plugin("OAuth2Accounts" => {
   
    $r->get('/connect/:provider' => sub {
     my $self = shift;
-    warn $self->session('id');
     return $self->redirect_to('connectprovider', {provider => $self->param('provider')}) unless $self->session('id');
     $self->redirect_to('questions');
   });
+
+  $r->get('/admin' => sub {
+    my $self = shift;
+    $self->render('admin');
+  });
```

Comment:
Of course! I seem to be forgetting the basics and instead relying on recycling code.